tools/centos.md

```diff
@@ -14,24 +14,34 @@ releases:
     eol: 2020-11-30
     latest: "6.10"
     link: https://wiki.centos.org/Manuals/ReleaseNotes/CentOS6.10
-  - releaseCycle: "CentOS 7"
+  - releaseCycle: "CentOS Linux 7"
     release: 2014-07-07
-    support: 2020-12-31
+    support: 2020-08-06
     eol: 2024-06-30
-    latest: "7 (2003)"
-    link: https://wiki.centos.org/Manuals/ReleaseNotes/CentOS7.2003
-  - releaseCycle: "CentOS 8"
+    latest: "7 (2009)"
+    link: https://wiki.centos.org/Manuals/ReleaseNotes/CentOS7.2009
+  - releaseCycle: "CentOS Linux 8"
     release: 2019-09-24
     support: 2021-12-31
     eol: 2021-12-31
-    latest: "8 (2004)"
+    latest: "8 (2105)"
     link: https://wiki.centos.org/Manuals/ReleaseNotes/CentOS8.2004
```

Comment:
The release notes link should also be updated. The rest is ok for me.

Reply:
Thanks for catching that.  Fixed now.